roles/openshift_repos/meta/main.yml

```diff
@@ -11,4 +11,5 @@ galaxy_info:
     - 7
   categories:
   - cloud
-dependencies: []
+dependencies:
+- role: openshift_inventory
```

Comment:
Yes, I've read that documentation before and still observed different behavior when we had the checks implemented using multiple roles. The role that was a common dependency got executed multiple times, despite of what the docs say... and I think we have the same problem with `openshift_facts`?